RatScanner/View/MainWindow.xaml

```diff
@@ -135,10 +147,10 @@
         <Separator/>
 
         <Grid Margin="0,5">
-          <Label Grid.Column="0" Content="Wiki Link"/>
-          <TextBlock Grid.Column="1">
-            <Hyperlink NavigateUri="{Binding WikiLink}" RequestNavigate="HyperlinkRequestNavigate" TextDecorations="">
-              <Label Content="Click to open"/>
+          <Label Grid.Column="0" Content="Links" VerticalAlignment="Center"/>
```

Comment:
`VerticalAlignment="Center"` is redundant here

Reply:
Removed